sympy/tensor/index_methods.py

```diff
@@ -24,6 +23,7 @@ class IndexConformanceException(Exception):
     pass
 
 
+@deprecated(useinstead='EinsteinSum', issue=9284)
 def _remove_repeated(inds):
```

Comment:
Is `_remove_repeated` used in any public function that's being deprecated?
If no, then @moorepants is right and `_remove_repeated` should simply be removed.
If yes, then I'd recommend going with (1) by @jcrist above, or with my proposal if adapting the doctest is more work than is worth throwing at a deprecated function. (@jcrist what's the use case where (2) is a good idea? My imagination is failing me.)